index.xml

```diff
@@ -8626,6 +8748,11 @@ context.setColorEnforcedRange(-1, 255, 256);</x:codeblock>
               <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>
               <span class='rfc2119'>MUST</span> be a subset of the interface's
               <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>.
+              Similarly, if <a class='xattr' href='#Exposed'>[Exposed]</a> appears on both a
```

Comment:
This isn't quite right, just like the existing interface text isn't quite right.  Consider this IDL:

    namespace foo {
      [Exposed=Worker] void func();
    };

This should be disallowed, because the namespace is only exposed on Window.  But the text you're adding does not apply, because the namespace itself doesn't have [Exposed] specified.

I think this text should go like this: "If [Exposed] appears on a namespace member, then the namespace member's exposure set MUST be a subset of the exposure set of the namespace or partial namespace it's a member of." or something along those lines.  And similar for interfaces.